packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -159,4 +128,4 @@ const ActionList = assignWithoutSideEffects(React.memo(_ActionList), {
 });
 
 export type { ActionListProps };
-export { ActionList, useActionListContext };
+export { ActionList };
```

Comment:
What if existing consumers were using useActionListContext since its exported?

Reply:
I guess it was only exported fir tests. Consumers cant really use it since they can't add custom components in ActionList